pkg/controller/backup_sync_controller.go

```diff
@@ -66,10 +65,10 @@ func NewBackupSyncController(
 	newPluginManager func(logrus.FieldLogger) clientmgmt.Manager,
 	logger logrus.FieldLogger,
 ) Interface {
-	if syncPeriod < time.Minute {
-		logger.Infof("Provided backup sync period %v is too short. Setting to 1 minute", syncPeriod)
+	if syncPeriod <= 0 {
```

Comment:
Doesn't this conditional keep the effective minimum sync time at 1 minute?

Reply:
No - you can explicitly specify a sync period of e.g. `15s`. This just defaults it to 1 minute if a 0/negative value was provided.